c2cgeoform/schema.py

```diff
@@ -25,6 +25,17 @@ def objectify(self, dict_, context=None):
         return super().objectify(dict_, context)
 
 
+class GeoFormManyToManySchemaNode(GeoFormSchemaNode):
```

Comment:
Not sure whether it is a good idea or not, but one could choose to factorize 
"""
class_ = self.inspector.class_ ...
return query.one_or_none()""""
between objectify and validator.